turbinia/evidence.py

```diff
@@ -242,11 +243,68 @@ def __init__(
     # TODO(aarontp): Make this an enum, or limited list
     self.encryption_type = encryption_type
     self.encryption_key = encryption_key
-    # self.local_path will be the encrypted path
     self.unencrypted_path = unencrypted_path
     super(EncryptedDisk, self).__init__(*args, **kwargs)
 
 
+class BitlockerDisk(EncryptedDisk):
+  """Bitlocker encrypted disk file evidence.
+
+  Attributes:
+    recovery_key: A string of the recovery key for this disk
+    password: A string of the password used for this disk. If recovery key
+        is used, this argument is ignored
+    unencrypted_path: A string to the unencrypted local path
+  """
+
+  def __init__(
+      self, recovery_key=None, password=None, encrypted_path=None,
+      unencrypted_path=None, *args, **kwargs):
+    """Initialization for Bitlocker disk evidence object"""
+    self.recovery_key = recovery_key
+    self.password = password
+    self.encrypted_path = encrypted_path
+    self.unencrypted_path = unencrypted_path
```

Comment:
I think we can re-use some of the attributes from `EncryptedDisk` here. `unencrypted_path` can just get passed through `super` and you could probably move `encrypted_path` there as well (or just re-use `local_path` from RawDisk)?